### ISSUE TITLE
NO AUTHORIZATION / NO CREDIT ???!!!!????!!!

### ISSUE THREAD

**Issue by jknppp** [PiSaucer/v1-windows-93-repo#5]
WHY ARE YOU PUBLISHING OUR SOURCE CODE ON GITHUB????
U MAD BRO????

**Comment by PiSaucer** [comment]
1.b I am mad that my school blocks the website.
1.b I am also mad that you didn't upload it yourself
2.a I am publishing the source to should you anyone can do
2.b Also I have a website where I run websites on my own server so I play unblock
2.c Also I think others would like to help improve your servers
3.a You got Credit in the windows93 in itself
3.b Everyone has authorization for your information

I will delete tomorrow because I reach the maximum deleted repo in one day. version 2 is delete thou.

**Comment by coreybruce** [comment]
Why are you so mad about it being shared? you should be sharing this amazing code not hording it to yourself, the best solution is to upload to your own repo for others to be able to play with, customize and more @jknppp

**Comment by coreybruce** [comment]
@PiSaucer Please don't delete it, lets all talk about this, if you wanted credit I am sure he could add the credit for your work

**Comment by PiSaucer** [comment]
I gave you credit if you only look at the readme.
Is this anything least I need to do so I don't have to delete the repo?

**Comment by coreybruce** [comment]
Why can't we contribute to your code and why cant this be considered a fork of Windows 93?

**Comment by jknppp** [comment]
Guys… you seem so sure of yourselves.
Have you wondered if there was a reason why we did not upload our code ourselves to github?
I am not a teenager, I am a person who has to work to survive and who has spent years developing a project for others without asking anyone for money. You have no idea what we've been through since the beginning of this project. You have no idea of the vultures against whom we have had to defend ourselves so that you can enjoy our work.
When you publish our code without our consent you are disrespectful and indirectly endangering us.
Regarding this repo, there is no discussion: you erase it, basta.
I will upload my code when I decide that it is ready to be published.

**Comment by coreybruce** [comment]
> Guys… you seem so sure of yourselves.
> Have you wondered if there was a reason why we did not upload our code ourselves to github?
> I am not a teenager, I am a person who has to work to survive and who has spent years developing a project for others without asking anyone for money. You have no idea what we've been through since the beginning of this project. You have no idea of the vultures against whom we have had to defend ourselves so that you can enjoy our work.
> When you publish our code without our consent you are disrespectful and indirectly endangering us.
> Regarding this repo, there is no discussion: you erase it, basta.
> I will upload my code when I decide that it is ready to be published.

Your right we may not know what you have been through but by it being on GitHub mean you won't have to do it all alone, when it's open source and on Github we as a community can help you to improve and expand the project with big numbers of contributes and growth of speed to continue fixing improving the code instead you guys doing it yourself.

A idea I had was this being a great base for a serious web based OS because the code, window manager and desktop are there and could expand into something amazing that could take you guys places in the tech world.

**Comment by PiSaucer** [comment]
so, delete the repo or not?

**Comment by coreybruce** [comment]
I don't know I mean I really don't want it to be removed nor do I see a issue if we want to share an help with his project. he hasn't replied back and given any real reason other than oh you didn't ask

**Comment by jknppp** [comment]
> so, delete the repo or not?
@PiSaucer  delete pls

> he hasn't replied back and given any real reason other than oh you didn't ask
@coreybruce do I have to write you an entire book about my reasons here? give me a break boi.

https://github.com/PiSaucer/v1-windows-93-repo/issues/5#issuecomment-477100944
I know what's open source thanks, and rn I don't want to talk with you about your ideas.
I'll publish my repo when I want to, not like this (in a raspi repo? wtf) and not now. 
Can you understand that or thot?

**Comment by coreybruce** [comment]
@jknppp I wasn't explaining what open source was I was saying that open sourcing it would be a great idea, I hope to see a repo from you guys at some point.

**Comment by DracoZ12** [comment]
This clearly isnt even "open source" since it doesnt even contain the source. You guys just used HTTrack.

**Comment by DracoZ12** [comment]
jknppp is obviously not jankenpopp.

**Comment by PhantomDoom741** [comment]
Uhh making different versions of Windows93 (aka win93 clones) is already possible, a repo isn't needed - you just need the right stuff, then you can make your version of Win93.

**Comment by Dakedres** [comment]
The problem is that this is the original source, only modified slightly to fit your use case. You can't copyright this, it was already property of Sys42.

**Comment by PiSaucer** [comment]
Yeah, I modified it a bit to fit my needs like adding a 404 and 403 html which wasn't part of Windows 93 version 1. I also modified the credits to get myself some credit. Somethings were also broked that I try to fix.

in the terms and agreements, you say
 
"You are free to:
Share — copy and redistribute the material in any medium or format 
Adapt — remix, transform, and build upon the material 
The licensor cannot revoke these freedoms as long as you follow the license terms. 
Under the following terms: 
Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use. 
NonCommercial — You may not use the material for commercial purposes. 
No additional restrictions — You may not apply legal terms or technological measures that legally restrict others from doing anything the license permits."

**Comment by DracoZ12** [comment]
He is still the owner of that code. If he doesn't want it here, just delete the repo. Done
